recipes/h5py/bld.bat

```diff
@@ -0,0 +1,5 @@
+"%PYTHON%" setup.py configure --hdf5="%LIBRARY_PREFIX%"
```

Comment:
Switched to using the [`configure`]( http://docs.h5py.org/en/latest/build.html#custom-installation ) option as setting the environment variable wasn't working on Windows (could have been doing it wrong). This seems a bit nicer and more consistent across platforms.